plugins/k8s/content.yaml

```diff
@@ -0,0 +1,99 @@
+title: Kubernetes
```

Comment:
can we change this to k8s? - we already have a plugin titled Kubernetes and it is confusing when searching :(

Reply:
Thanks for your reply, but I have a confusion.
When I tested locally, I found that search has nothing to do with tags, only with title. 
So in order to make it better discovered, can I change it to `Kubernetes/k8s` or `Kubernetes Plugin`.